api/proto/printable.proto

```diff
@@ -49,6 +49,7 @@ message TransferPayload {
 }
 
 // Message encoding information required to locate a TxOut,
+//
```

Comment:
this empty line alleviates a clippy error for first doc line being too long in the generated code.